### ISSUE TITLE
RAL Stratum 1 issue

### ISSUE THREAD

**Issue by StevenCTimm** [DUNE/dist-comp#109]
We have 3 sites where fifeuser4.opensciencegrid.org repo is way behind, namely 

Jake Calcutt
  [2:42 AM](https://dunescience.slack.com/archives/C7Z6Z9BBP/p1696405367024829)
Here is info about the RCDS issue I'm seeing
Hi Jacob,
checking RCDS logs we know the code tarball was properly published on RCDS CVMFS volumes (thanks Shreyas).
This job was running at Manchester site, and checking CVMFS revisions for RCDS repositories it looks like at the site the revision are left quite behind.
According to Dave D. this is due to an already known RAL stratum1 problem. I'll try to ping them to check what the status is.
 
In the meantime I'd suggest to avoid to submit jobs that require RCDS at Manchester site.
 
-Vito
I've found this issue on the following sites:
Manchester
SGrid
SGridOxford

**Comment by StevenCTimm** [comment]
E-mailed Dave Dykstra to ask for more details.  Is it time to put another dune-focused stratum 1 in the UK?

Steve

**Comment by Andrew-McNab-UK** [comment]
[ALICE ](https://ggus.eu/index.php?mode=ticket_info&ticket_id=163470), [CMS](https://ggus.eu/index.php?mode=ticket_info&ticket_id=163483) and [LHCb ](https://ggus.eu/index.php?mode=ticket_info&ticket_id=163484) tickets for RAL Tier-1 about this.

**Comment by Andrew-McNab-UK** [comment]
RAL have just now sent this round: 

“The CVMFS Stratum-1 service at RAL is experiencing problems since the 29th of September. Since then, the repositories are not being updated and, therefore, the service is distributing out of date content. 
This is causing a lot of failures on running jobs, as the CVMFS client is not aware of that.

We are currently investigating the root cause of the problem. Meanwhile, we have set the service offline. The hope is that, when the clients are unable to contact it, they will try with another Stratum-1 with up-to-date content.

As our Frontier-Squid most probably also has out of date content, we have disabled it as well. 

As soon as the problem gets fixed, we will re-enable the services and communicate it accordingly. “
